### PR TITLE
Improve the README.md to make it clear and simple to follow

### PR DESCRIPTION
**Pull request by rpsene**
Signed-off-by: Rafael Peria de Sene <rpsene@br.ibm.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/106#" title="Author self-approved">rpsene</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **prajyot-parab**
You can assign the PR to them by writing `/assign @prajyot-parab` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: README.md [hunk 1]
```diff
@@ -1,48 +1,32 @@
...
-- https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started
-- https://www.youtube.com/watch?v=RywSfXT_LLs
-- https://www.youtube.com/playlist?list=PLVrJaTKVPbKM_9HU8fm4QsklgzLGUwFpv
+# PowerVS Preparation
 
+You need to handle an initial preparation in your PowerVS instance to successfuly execute this automation. Follow this [guide](docs/ocp_prereqs_powervs.md) to get it done. To get started with PowerVS, [click here](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started). You can also learn more about PowerVS by watching some YouTube videos by clicking [here](https://www.youtube.com/watch?v=RywSfXT_LLs) and [here](https://www.youtube.com/playlist?list=PLVrJaTKVPbKM_9HU8fm4QsklgzLGUwFpv).
 
-:heavy_exclamation_mark: **Ensure that you're working from the desired release-specific version of this `README` as you deploy by first selecting the appropriate branch from the drop-down above. At the time of this writing, the latest release is 4.5: [README](https://github.com/ocp-power-automation/ocp4-upi-powervs/tree/release-4.5). The `README` document located in the `master` branch changes frequently and should not be considered final.**
```
Comment (bpradipt) [inline README.md:25]: Please do not delete this text for now. This is specifically added for first time users to try stable release.

**Comment by bpradipt** [comment]
@clnperez can you please review the changes..

**Comment by yussufsh** [comment]
/approve
Look simpler and better for me.

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/106#" title="Author self-approved">rpsene</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/106#issuecomment-703077884" title="Approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by clnperez** [comment]
Was out for a few days. Will review ASAP @bpradipt.

**Comment by bpradipt** [comment]
@rpsene will need a rebase.. 
The doc looks good to me.. Will wait for comments from @clnperez

File: README.md [hunk 2]
```diff
@@ -1,48 +1,34 @@
...
 
-- [Table of Contents](#table-of-contents)
-  - [Introduction](#introduction)
-  - [Automation Host Prerequisites](#automation-host-prerequisites)
-  - [PowerVS Prerequisites](#powervs-prerequisites)
-  - [OCP Install](#ocp-install)
-  - [Contributing](#contributing)
+# Welcome
 
+This repository contains Terraform templates, configurations and automation which leverages the [helpernode ansible playbook](https://github.com/RedHatOfficial/ocp4-helpernode) to assist on deploying [OpenShift Container Platform (OCP) 4.x](https://www.openshift.com/products/container-platform) on [IBM® Power Systems™ Virtual Server on IBM Cloud](https://www.ibm.com/cloud/power-virtual-server).
```
Comment (clnperez) [inline README.md:6]: one nit: to assist *in* deploying

File: README.md [hunk 3]
```diff
@@ -1,48 +1,34 @@
...
+:heavy_exclamation_mark: for any bugs, enhancement, requests, fixes, etc. [open a new GitHub issue](https://github.com/ocp-power-automation/ocp4-upi-powervs/issues/new) and ensure it is not duplicated.
 
-:heavy_exclamation_mark: *For bugs/enhancement requests etc. please open a GitHub issue*
+:heavy_exclamation_mark: [checkout the latest stable version](https://github.com/ocp-power-automation/ocp4-upi-powervs/tree/release-4.5).
 
-For general PowerVS usage instructions please refer to the following links:
-- https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started
-- https://www.youtube.com/watch?v=RywSfXT_LLs
-- https://www.youtube.com/playlist?list=PLVrJaTKVPbKM_9HU8fm4QsklgzLGUwFpv
+:heavy_exclamation_mark: ensure that you're working from the desired release-specific version of this `README` as you deploy by first selecting the appropriate branch from the drop-down above. At the time of this writing, the latest release is 4.5: [README](https://github.com/ocp-power-automation/ocp4-upi-powervs/tree/release-4.5). The `README` document located in the `master` branch changes frequently and should not be considered final.
```
Comment (clnperez) [inline README.md:16]: nit: capitalize the first letters in sentences "For any bugs..." "Check ou  the latest..." "Ensure that you're..."

File: README.md [hunk 4]
```diff
@@ -1,48 +1,34 @@
...
 
+# PowerVS Preparation
 
-:heavy_exclamation_mark: **Ensure that you're working from the desired release-specific version of this `README` as you deploy by first selecting the appropriate branch from the drop-down above. At the time of this writing, the latest release is 4.5: [README](https://github.com/ocp-power-automation/ocp4-upi-powervs/tree/release-4.5). The `README` document located in the `master` branch changes frequently and should not be considered final.**
+You need to handle an initial preparation in your PowerVS instance to successfuly execute this automation. Follow this [guide](docs/ocp_prereqs_powervs.md) to get it done. To get started with PowerVS, [click here](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started). You can also learn more about PowerVS by watching some YouTube videos by clicking [here](https://www.youtube.com/watch?v=RywSfXT_LLs) and [here](https://www.youtube.com/playlist?list=PLVrJaTKVPbKM_9HU8fm4QsklgzLGUwFpv).
```
Comment (clnperez) [inline README.md:20]: Suggested verbiage: "You first need to create and configure a PowerVS Intance in IBM Cloud before running this automation. You may follow [this guide](docs/ocp_prereqs_powervs.md) for instructions."

The other information, I think, should be moved to the PowerVS pre-reqs doc.

File: README.md [hunk 5]
```diff
@@ -1,48 +1,34 @@
...
 
+# Automation Host Preparation
 
-## Automation Host Prerequisites
+You can run this automation from any system (laptop, virtual machine, etc.) with internet access. It was tested on the following operating systems:
```
Comment (clnperez) [inline README.md:24]: "It has been tested on the following..."

**Review by clnperez — CHANGES_REQUESTED** [review]
Mostly small grammar comments. The only real suggestion is to put the PowerVS info into the PowerVS pre-req doc instead of here.

**Comment by bpradipt** [comment]
@rpsene can you please rebase and send the updated PR.

**Comment by rpsene** [comment]
@bpradipt done!

**Comment by clnperez** [comment]
@rpsene can you move this "about powervs" powervs detail to the prereq doc? 

`
To get started with PowerVS, [click here](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started). You can also learn more about PowerVS by watching some YouTube videos by clicking [here](https://www.youtube.com/watch?v=RywSfXT_LLs) and [here](https://www.youtube.com/playlist?list=PLVrJaTKVPbKM_9HU8fm4QsklgzLGUwFpv).
`

It's good info. I just think having it all listed there is distracting.

**Comment by rpsene** [comment]
> @rpsene can you move this "about powervs" powervs detail to the prereq doc?
> 
> `To get started with PowerVS, [click here](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started). You can also learn more about PowerVS by watching some YouTube videos by clicking [here](https://www.youtube.com/watch?v=RywSfXT_LLs) and [here](https://www.youtube.com/playlist?list=PLVrJaTKVPbKM_9HU8fm4QsklgzLGUwFpv).`
> 
> It's good info. I just think having it all listed there is distracting.

Why distracting if we have users that have difficult getting started with the basic steps on PowerVS? I would like to keep it in the front page so it is easy to find.

**Comment by clnperez** [comment]
To me, it seems like too much is going on (too many links and places to diverge into) in that one little snippet. My brain would read this easier if we point users to the more in-depth PowerVS guide (as is done in the first sentence), and there include the additional info about PowerVS.

**Comment by bpradipt** [comment]
Closing this as this is stale PR.

**Pull request closed without merging**